ui/lib/home/page/home_page.dart

```diff
@@ -27,10 +27,10 @@ class HomePage extends StatelessWidget {
   Widget _buildSideBar(BuildContext context) {
     return SizedBox.fromSize(
       size: Size.fromWidth(Sizing.homeSideBarWidth),
-      child: Column(
+      child: ListView(
```

Comment:
use ListView.builder for performance gains here. Regular ListView isn't much better than SingleChildScrollView.